ibmcloud/image/import.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+# import.sh takes a podvm docker image reference and a cloud region
+# and creates a ibmcloud vpc image of the qcow2 image.
+# Requires IBMCLOUD_API_KEY to be set.
+# Will default to first bucket it can find, by specifying
+# --bucket & --instance to use a particular bucket
+
+error(){
+    echo $1 1>&2 && exit 1
+}
+
+script_dir=$(dirname "$0")
+
+function usage() {
+    echo "Usage: $0 <docker-image> <vpc-region> [--bucket <name> --region <cos-region> --instance <cos-instance> --endpoint <cos-endpoint> --api <cloud-endpoint>]"
+}
+
+image=$1
+region=$2
+bucket=
+bucket_region=$region
+instance=
+endpoint=
+api=https://cloud.ibm.com
+
+shift 2
+while (( "$#" )); do
+    case "$1" in
+        --bucket) bucket=$2 ;;
+        --instance) instance=$2 ;;
+        --endpoint) endpoint=$2 ;;
+        --region) bucket_region=$2 ;;
+        --api) api=$2 ;;
+        --help) usage; exit 0 ;;
+        *)      usage 1>&2; exit 1;;
+    esac
+    shift 2
+done
+
+if [[ -z "${image-}" || -z "${region-}"  ]]; then
+    usage 1>&2
+    exit 1
+fi
+
+[ -z "$IBMCLOUD_API_KEY" ] && error "IBMCLOUD_API_KEY is not set"
+
+IBMCLOUD_API_ENDPOINT="$api" $script_dir/login.sh
+ibmcloud target -r "$region"
```

Comment:
That would be fine for now. Hopefully once we'll solidified the operator work we'll be able to massively reduce the current doc into a better flow and then it will fit there better. I'm think something like:
- Optionally Create a podvm image
  - This would include the section explaining/linking to the authenticated registry support set-up
- Upload a podvm image (this PRs work)
- Create a cluster
- Deploy the operator
- Run the tests

PS: Sorry - the doc plan obviously is out of scope for this PR, I just started doing the typing equivalent of thinking out loud!